packages/core/type-definitions/combinator/merge.d.ts

```diff
@@ -1,4 +1,4 @@
 import { Stream } from '@most/types';
 
-export function merge<A>(...streams: Array<Stream<A>>): Stream<A>;
+export function merge<A>(a: Stream<A>, b: Stream<A>): Stream<A>;
```

Comment:
Whatever names we pick for these two streams in the `merge` flow type refs, let's use the same names here.